app/forms/authorisedOfficials/AuthorisedOfficialsPhoneNumberFormProvider.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.authorisedOfficials
+
+import forms.mappings.Mappings
+import javax.inject.Inject
+import models.AuthorisedOfficialsPhoneNumber
+import play.api.data.Form
+import play.api.data.Forms._
+
+class AuthorisedOfficialsPhoneNumberFormProvider @Inject() extends Mappings {
+
+  val validateTelephoneNumber = """^\+?(?:\s*\d){10,13}$"""
```

Comment:
Plz change is variable to private like **private[authorisedOfficials] val validateTelephoneNumber**

Reply:
Updated